lib/ansible/module_utils/vmware.py

```diff
@@ -304,10 +468,18 @@ def list_snapshots(vm):
 def vmware_argument_spec():
 
     return dict(
-        hostname=dict(type='str', required=True),
-        username=dict(type='str', aliases=['user', 'admin'], required=True),
-        password=dict(type='str', aliases=['pass', 'pwd'], required=True, no_log=True),
-        validate_certs=dict(type='bool', required=False, default=True),
+        hostname=dict(
+            type='str', required=True
+        ),
+        username=dict(
+            type='str', aliases=['user', 'admin'], required=True
+        ),
+        password=dict(
+            type='str', aliases=['pass', 'pwd'], required=True, no_log=True
+        ),
+        validate_certs=dict(
+            type='bool', required=False, default=True
+        ),
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.